### PR TITLE
Update OWNERS for realignment

### PR DESCRIPTION
**Pull request by mjlshen**

### CODE REVIEW

File: boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES [hunk 1]
```diff
@@ -29,8 +29,9 @@ aliases:
     - devppratik
   srep-functional-team-orange:
     - bng0y
+    - bergmannf
     - typeid
-    - Makdaam
+    - mrWinston
     - Nikokolas3270
     - ninabauer
```
Comment (typeid) [inline boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES:36]: Could remove Nina as well, she moved to app-sre

File: boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES [hunk 2]
```diff
@@ -29,8 +29,9 @@ aliases:
     - devppratik
   srep-functional-team-orange:
     - bng0y
+    - bergmannf
     - typeid
-    - Makdaam
```
Comment (typeid) [inline boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES:33]: Not sure about this: @Makdaam can you confirm?

**Comment by typeid** [comment]
We should probably add the step of adding to the team alias here to onboarding.

File: boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES (hunk 2 above)
Comment (mjlshen) [reply boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES:33]: I thought @Makdaam's off to a new team as well :D

**Pull request closed without merging**